m2cgen/assemblers/boosting.py

```diff
@@ -100,6 +119,33 @@ def _assemble_child_tree(self, tree, child_id):
         assert False, "Unexpected child ID {}".format(child_id)
 
 
+class LightGBMModelAssembler(BaseBoostingAssembler):
+
+    classifier_name = "LGBMClassifier"
+
+    def __init__(self, model):
+        model_dump = model.booster_.dump_model()
+        trees = model_dump["tree_info"]
+
+        super().__init__(model, trees)
+
+    def _assemble_tree(self, tree):
+        if "tree_structure" in tree:
```

Comment:
Apparently each `tree` object should container either `tree_structure` or `leaf_value` otherwise something went very wrong. Can we express this with the `assert`?

Reply:
Done, thanks!